packages/spor-theme-react/src/components/button.ts

```diff
@@ -0,0 +1,168 @@
+import type {
+  SystemStyleFunction,
+  SystemStyleObject,
+} from "@chakra-ui/theme-tools";
+
+const baseStyle: SystemStyleObject = {
+  border: 0,
+  borderRadius: "xl",
+  fontWeight: "bold",
+  whiteSpace: "nowrap",
+  transitionProperty: "common",
+  transitionDuration: "normal",
+  px: 3,
+  _focus: {
+    boxShadow: 0,
+    outline: 0,
+  },
+  _disabled: {
+    cursor: "not-allowed",
+    boxShadow: "none",
+    backgroundColor: "alias.silver",
```

Comment:
these `"alias.silver"` values are fetched from the theme definition of colors, which you can find [here](https://github.com/nsbno/spor/blob/main/packages/spor-theme-react/src/foundations/colors.ts)